express-hello-world/index.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const app = express();
+
+app.use(function (req, res, next) {
+  // console.log('Request Type:', req.method);
+  res.send('This is a response string');
+  next();
+});
```

Comment:
`next` should only be called if the current middleware isn't sending a response.